source/localizable/smart_cart/orders_api.html.md.erb

```diff
@@ -158,6 +158,61 @@ also get a `422` HTTP status code (Unprocessable Entity), with the following bod
 
 [More on API error handling](/api/v3/#error-handling)
 
+## Test requests (sandbox)
+
+The following d​emo order codes​ are available on production for testing reasons, andrepresent different order states,
```

Comment:
We may want to add a "warning" yellow box (see above/bellow), noting that it is safe to accept/reject these "orders". eg:

> It is safe to issue "accept" and "reject" API calls for the following demo order codes multiple times for testing, since no actions on them would persist any data. 

WDYT?

Reply:
I think this is a good note to add.